_translator-files/po/de/Tips-Tricks-More.po

```diff
@@ -45,116 +47,118 @@ msgid ""
-msgstr ""
+msgstr "Jamulus-Benutzer [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) hat eine große Menge an Informationen zu [Band Proben aus der Ferne](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc) zusammengestellt, die Themen wie Hardware- und Softwarekonfiguration einschließlich Beispielen und Ratschlägen für Neulinge auf diesem Gebiet abdecken. Er enthält auch einen Abschnitt über Jamulus im Vergleich zu anderen Lösungen."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:25
 msgid "## Using the Jamulus Client"
-msgstr ""
+msgstr "## Verwendung des Jamulus-Client"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:27
 msgid "### Have a undisturbed session on any Server"
-msgstr ""
+msgstr "### Eine ungestörte Sitzung auf einem beliebigen Server einrichten"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:29
 msgid "You can have a \"private\", undisturbed session with other people on any Server, including publically listed Servers, by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information)."
-msgstr ""
+msgstr "Du kannst eine \"private\", ungestörte Sitzung mit anderen Personen auf einem beliebigen Server, einschließlich öffentlich gelisteten Servern, abhalten, indem sich alle Teilnehmer auf SOLO schalten. Du wirst dann nicht in der Lage sein, andere Personen zu hören, wenn diese den selben Server betreten (beachte, dass dies andere nicht daran hindert, dich zu hören, die Chat-Funktion zu benutzen oder deine Profilinformationen zu sehen)."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:31
 msgid "### Using Jamulus audio in Zoom (or other) meeting apps"
-msgstr ""
+msgstr "### Jamulus Audio in Zoom (oder anderen Meeting-Apps) verwenden"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:33
 msgid "Several users have reported success allowing a \"virtual audience\" for a Jamulus session by using [JACK audio](https://jackaudio.org) to route the Jamulus signal through JackRouter to the target application (in this case, Zoom meetings)."
-msgstr ""
+msgstr "Mehrere Benutzer haben berichtet, dass es ihnen gelungen ist, ein \"virtuelles Publikum\" für eine Jamulus-Sitzung zu schaffen, indem sie [JACK Audio](https://jackaudio.org) verwendet haben, um das Jamulus-Signal über JackRouter an die Zielanwendung (in diesem Fall Zoom-Meetings) zu leiten."
```

Comment:
Not related: This should link to my KB article. 

@gilgongo ?

Reply:
The KB stuff is always a bit tricky :-) The basic purpose of the KB (other than creating a "content cloud" to raise our profile in Google searches) is to have content that the core team doesn't have time to maintain. By implication it can therefore go out of date, so linking to it from the "official" docs isn't a good idea. The grey area is Tips & Tricks of course...

Reply:
@ann0see & @gilgongo : You decide :)

Reply:
I'd prefer we didn't link out of the official docs to the KB myself, and if we think we should on a case-by-case basis, we need to consider why.